v2v-helper/Makefile

```diff
@@ -1,8 +1,15 @@
 SHELL :=  /usr/bin/env bash
+PKG_CONFIG_PATH ?= /usr/local/lib/pkgconfig
+CGO_CFLAGS := $(shell PKG_CONFIG_PATH=$(PKG_CONFIG_PATH) pkg-config --cflags libnbd)
+CGO_LDFLAGS := $(shell PKG_CONFIG_PATH=$(PKG_CONFIG_PATH) pkg-config --libs libnbd)
 
 LDFLAGS += "-s -w -X github.com/platform9/vjailbreak/v2v-helper/pkg/version.Version=$(RELEASE_VERSION)"
 BUILD_OPTIONS += -ldflags=$(LDFLAGS) -buildvcs=false
 
+export CGO_CFLAGS
+export CGO_LDFLAGS
+export LD_LIBRARY_PATH := /usr/local/lib:$(LD_LIBRARY_PATH)
```

Comment:
use the LDFLAGS on line 6 and add this path to that string. and use LDFLAGS in the compilation.